node/test/unit/tessel.js

```diff
@@ -1261,14 +1277,14 @@ exports['Tessel.Pin'] = {
       this.a.pin[pinIndex].on('rise', spy);
       this.b.pin[pinIndex].on('rise', spy);
 
-      test.equal(this.a.pin[pinIndex].interruptMode, 'rise');
-      test.equal(this.b.pin[pinIndex].interruptMode, 'rise');
+      test.equal(this.a.pin[pinIndex].interruptMode, 'change');
+      test.equal(this.b.pin[pinIndex].interruptMode, 'change');
 
       // Simulate receipt of pin state changes
-      this.a.sock.read.returns(new Buffer([REPLY.ASYNC_PIN_CHANGE_N + pinIndex]));
+      this.a.sock.read.returns(new Buffer([(REPLY.ASYNC_PIN_CHANGE_N + pinIndex) | (1 << 3)]));
       this.a.sock.emit('readable');
 
-      this.b.sock.read.returns(new Buffer([REPLY.ASYNC_PIN_CHANGE_N + pinIndex]));
+      this.b.sock.read.returns(new Buffer([(REPLY.ASYNC_PIN_CHANGE_N + pinIndex) | (1 << 3)]));
       this.b.sock.emit('readable');
```

Comment:
I tested 'accel-mma84', 'realy-mono', 'climate-si7020' and 'sdcard' using the latest revisions for each from npm. They all work fine with the exception of 'sdcard', but that is because it looks like it hasn't been updated for T2... it still uses synchronous `.read()`.